README.md

```diff
@@ -1 +1,67 @@
-# my-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Dyary's Portfolio
+
+> this is my portfolio website. It includes info about me, how to contact me and my achivemets and creations as a web developer.
+
+![screenshot](https://i.imgur.com/vWXwTaR.png)
+
+look thorugh the website to see my work and contact me if need.
+
+## Built With
+
+- HTML and CSS
+- VS Code, Chrome DevTools
+
+## Live Demo
+
+[Live Demo Link](https://i.imgur.com/vWXwTaR.png)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+install VS Code. and Chrome
+
+### Setup
+
+download the repositoyr using git clone and install live-server VS Code Extention
+
+### Install
+
+cd into the repository folder on your local machine.
+use the command live-serve in the VS code terminal
+
+### Deployment
+
+run live-server.
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/DyaryRaoof)
+- Twitter: [@twitterhandle](https://twitter.com/DyaryRaoof)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/dyary-raoof-baiz-agha-b79444155)
+
```

Comment:
Instead of 👤 **Author1** you can add 👤 **Your_Name** to have a more impressive README file.
Instead - GitHub: [@githubhandle](https://github.com/DyaryRaoof) you can change to - GitHub: [@Your_Name_SocialMediaName](https://github.com/DyaryRaoof) to have a more impressive README file.

- [ ] [OPTIONAL] Make these changes for this and for your next projects to have a more professional README file 😉